### PR TITLE
Draft: DEV-31765 acrolinxLive flag

### PR DESCRIPTION
**Pull request by replrep**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -499,6 +499,10 @@ writing guide or a Checking Profile. If Sublanguages are activated, they're fold
 For each guidance profile, the server provides information about the language, activated goals, and term sets. The client may use this
 for filtering purposes, but they can only select one (complete) guidance profile for checking (for example, not deselect goals).
 
+If the server runs in Targets Mode, the guidance profiles contain an additional flag 'acrolinxLive'. This flag is true iff modern reuse is
```
Comment (rbarbey) [inline apiary.apib:502]: I guess this new flag should be wrapped in backticks to show it as code in the docs?

```suggestion
If the server runs in Targets Mode, the guidance profiles contain an additional flag `acrolinxLive`. This flag is true iff modern reuse is
```

File: apiary.apib [hunk 2]
```diff
@@ -499,6 +499,10 @@ writing guide or a Checking Profile. If Sublanguages are activated, they're fold
...
+enabled on the server, the current user has the document checking privilege and the guidance profile contains an active modern reuse guideline with
+at least one reuse set configured.
```
Comment (rbarbey) [inline apiary.apib:504]: Maybe it make this section more readable if all the conditions for this flag where part of a list? @madeline-brennan what do you think?

**Review by rbarbey — CHANGES_REQUESTED** [review]

File: apiary.apib (hunk 2 above)
Comment (madeline-brennan) [reply apiary.apib:504]: Yes, I think so. I'd propose: 
```suggestion
If Targets mode is active, there is an additional flag called 'acrolinxLive'. This flag applies if the following is true:

- Modern Reuse is enabled on the server.
- The current user has the "Check documents" privilege.
- The selected Target contains the Approved Messaging goal with at least one Reuse set is configured.
```

Comment (oliveoyl) [reply apiary.apib:504]: I'd propose these typo tweaks.

```suggestion
If Targets mode is active, the response contains a flag called 'acrolinxLive'. This flag applies if the following 3 conditions are true:
- Reuse is enabled on the platform.
- The current user has the "Check documents" privilege.
- The selected Target contains the Approved Messaging goal with at least one Reuse set configured.```

**Comment by robertgrabowski** [comment]
(Note: This PR is intentionally a draft - please don't merge until we review, change and extend the Acrolinx Live API in mid-2023.)

**Comment by hjohannsen** [comment]
@replrep, I renamed the old `developer` branch to `develop` old and created a fresh `develop`-branch from `main`. Because the old branch kept causing weird merge issues. 

Please rebase the changes in this MR on the new `develop`-branch and direct your PR to _that_ branch.

Thanks and sorry for the inconvenience!

**Pull request closed without merging**